docs/log_collection.md

```diff
@@ -26,61 +24,61 @@ Send logs to Datadog from your Android applications with [Datadog's `dd-sdk-andr
 
 2. Initialize the library with your application context, tracking consent, and the [Datadog client token][2] and Application ID generated when you create a new RUM application in the Datadog UI (see [Getting Started with Android RUM Collection][6] for more information). For security reasons, you must use a client token: you cannot use [Datadog API keys][3] to configure the `dd-sdk-android` library as they would be exposed client-side in the Android application APK byte code. For more information about setting up a client token, see the [client token documentation][2]:
 
-    {{< tabs >}}
-    {{% tab "US" %}}
-```kotlin
-class SampleApplication : Application() {
-    override fun onCreate() {
-        super.onCreate()
-
-        val config = DatadogConfig.Builder("<CLIENT_TOKEN>", "<ENVIRONMENT_NAME>", "<APPLICATION_ID>")
-                        .build()
-        Datadog.initialize(this, trackingConsent, config)
+   {{< tabs >}}
+   {{% tab "US" %}}
+   ```kotlin
+    class SampleApplication : Application() {
+        override fun onCreate() {
+            super.onCreate()
+            val configuration = Configuration.Builder().build()
+            val credentials = Credentials(<CLIENT_TOKEN>,<ENV_NAME>,<APP_VARIANT_NAME>,<APPLICATION_ID>)
+            Datadog.initialize(this, credentials, configuration, trackingConsent)
+        }
     }
-}
-```
-    {{% /tab %}}
-    {{% tab "EU" %}}
-```kotlin
-class SampleApplication : Application() {
-    override fun onCreate() {
-        super.onCreate()
-
-        val config = DatadogConfig.Builder("<CLIENT_TOKEN>", "<ENVIRONMENT_NAME>", "<APPLICATION_ID>")
-                        .useEUEndpoints()
-                        .build()
-        Datadog.initialize(this, trackingConsent, config)
-    }
-}
-```
-    {{% /tab %}}
-    {{< /tabs >}}
-    
-    To be compliant with the GDPR regulation, the SDK requires the tracking consent value at initialization.
-    The tracking consent can be one of the following values:
-    * `TrackingConsent.PENDING`: The SDK starts collecting and batching the data but does not send it to the data
+   ```
+   {{% /tab %}}
+   {{% tab "EU" %}}
+   ```kotlin
+   class SampleApplication : Application() {
+       override fun onCreate() {
+          super.onCreate()
+          val configuration = Configuration.Builder()
+             .useEUEndpoints()
+             .build()
+          val credentials = Credentials(<CLIENT_TOKEN>,<ENV_NAME>,<APP_VARIANT_NAME>,<APPLICATION_ID>)
```

Comment:
here we probably need to add also the `<SERVICE_NAME>` placeholder to make the user aware that he can actually override the default value. I am afraid because this information is duplicated in all the `.md` files we will have to do this everywhere :(.
```suggestion
          val credentials = Credentials(<CLIENT_TOKEN>,<ENV_NAME>,<APP_VARIANT_NAME>,<APPLICATION_ID>,<SERVICE_NAME>)
```
The `SERVICE_NAME` is optional though so I am not sure if we should add it in the docs or not. @xgouchet what do you think ?

Reply:
Actually no, I'd rather not make it too obvious. If it appears here, people might think they need to change it. And if they change it there, they'll need to also change it in the Gradle Plugin config, which increase the possibility of misshaps and support cases opened.